messages/README.md

```diff
@@ -0,0 +1 @@
+This directory holds emails or similar repeated or "form" emails that are part of some Astropy-specific process.  E.g., emails to people with new roles.
```

Comment:
I suggest:
This directory holds emails or similar repeated "forms" that are part of some Astropy-specific process, e.g., emails to people with new roles.